### ISSUE TITLE
Synchronous scrolling

### ISSUE THREAD

**Issue by karan173** [CJTozer/SublimeDiffView#62]
When scrolling the diff for a file, it would be really nice if both the views for the file can scroll together (like in diff viewers).

**Comment by CJTozer** [comment]
Yes, this is a nice idea - I've not checked for a while but when I developed this initially, the SublimeText API didn't make it easy to do this.  Things may have moved on, I may well check back and see what design options I have...

**Comment by pinheadmz** [comment]
Ooooh bump!
